test/functional/testlib.bash

```diff
@@ -1,6 +1,3 @@
-#!/usr/bin/bash
```

Comment:
if you remove this, shellcheck won't work

Reply:
my bad, it was a mistake